benchmarks/auth/python/Dockerfile.pypy

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Remove all dockerfiles but the .python one. Rename this one to only `Dockerfile`